modules/shortcodes/polldaddy.php

```diff
@@ -573,7 +573,7 @@ function polldaddy_link( $content ) {
 	add_filter( 'the_content_rss', 'polldaddy_link', 1 );
 
 	/** This filter is documented in modules/shortcodes/youtube.php */
-	if ( apply_filters( 'jetpack_comments_allow_oembed', get_option( 'embed_autourls' ) ) ) {
+	if ( apply_filters( 'jetpack_comments_allow_oembed', true ) ) {
```

Comment:
> we could use the same pattern as for the other options, and set a default value

Then every site would have an option set that WordPress doesn't even need anymore :)

I've changed my opinion from my original stance. I now think the current code in this PR (removing the call to `get_option()`) is fine even though it breaks backward compatibility.

Unless @jeherve, @kraftbj, @samhotchkiss, or some other Jetpack person disagrees, let's stick with your original idea.